### PR TITLE
Sample app

### LINKED ISSUE #10: Create sample app
Create a sample app showcasing:
* Usage of the library in a standard case
* Testing an animation using both `MockBlend` and `ImmediateBlend`

Moving forward we can:
* Make gifs using this app.
* Extend to show how to e.g. animate a custom value.

### PR DESCRIPTION
**Pull request by ryanmoelter**
Resolves #10. Add a sample app as an example of how to use and test with Blend.

* Create sample app
  * Show creating an animation
  * Show testing an animation with both `MockBlend` and `ImmediateBlend`

### CODE REVIEW

File: blend-sample-app/src/test/java/com/wealthfront/blend/sample/ExampleUnitTest.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.wealthfront.blend.sample
```
Comment (narenkmanoharan) [inline blend-sample-app/src/test/java/com/wealthfront/blend/sample/ExampleUnitTest.kt:1]: Should we remove this file?

File: blend-sample-app/src/androidTest/java/com/wealthfront/blend/sample/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.wealthfront.blend.sample
+
```
Comment (narenkmanoharan) [inline blend-sample-app/src/androidTest/java/com/wealthfront/blend/sample/ExampleInstrumentedTest.kt:2]: this file too. We should probably remove it.

File: blend-sample-app/src/test/java/com/wealthfront/blend/sample/ExampleUnitTest.kt (hunk 1 above)
Comment (ryanmoelter) [reply blend-sample-app/src/test/java/com/wealthfront/blend/sample/ExampleUnitTest.kt]: Done!

File: blend-sample-app/src/androidTest/java/com/wealthfront/blend/sample/ExampleInstrumentedTest.kt (hunk 2 above)
Comment (ryanmoelter) [reply blend-sample-app/src/androidTest/java/com/wealthfront/blend/sample/ExampleInstrumentedTest.kt]: Done!

**Pull request merged**